transports/quic/src/transport.rs

```diff
@@ -0,0 +1,350 @@
+use crate::crypto::Crypto;
+use crate::endpoint::{EndpointConfig, TransportChannel};
+use crate::muxer::QuicMuxer;
+use crate::{QuicConfig, QuicError};
+use ed25519_dalek::PublicKey;
```

Comment:
Why did you use ed25519_dalek::PublicKey instead of libp2p::identity::ed25519::PublicKey?